.ci/scripts/distribution/it-java.sh

```diff
@@ -17,7 +29,7 @@ if grep -q "\[WARNING\] Flakes:" ${tmpfile}; then
 
   awk '/^\[WARNING\] Flakes:.*$/{flag=1}/^\[ERROR\] Tests run:.*Flakes: [0-9]*$/{print;flag=0}flag' ${tmpfile} > ${tmpfile2}
 
-  grep "\[ERROR\]   Run 1: " ${tmpfile} | awk '{print $4}' >> ./target/FlakyTests.txt
+  grep "\[ERROR\]   Run 1: " ${tmpfile2} | awk '{print $4}' >> FlakyTests.txt
```

Comment:
@npepinpe don't forget this change then for the other scripts.

Reply:
I think on the main agent it doesn't fail because we build the root project, whereas on the IT we only build specific modules, but I can gladly change it.

Reply:
You can confirm this by removing your `./target` folder from your build and running verify for specific modules, it won't build it. But if you run for the project it will appear. Why it was however sometimes appearing for IT as well is a mystery to me.